pkg/imageengine/buildah/init.go

```diff
@@ -86,6 +121,11 @@ func initBuildah() error {
 		return err
 	}
 
+	storageAbsPath := "/etc/containers/storage.conf"
```

Comment:
do we need to mark storageAbsPath as a const ?

Reply:
It could be the const. But currently I think this will be used in one place only. I guess make it a temporary var is acceptable.